### PR TITLE
hilt and sever fixes

### PR DESCRIPTION
**Pull request by ezrn**
- Fixed sever and hilt smash sounds + logic
- Fixed sever cooldown not properly scaling
- Reach distance also fixed(?)

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have tested my changes.

### CODE REVIEW

**Comment by StudentAlleg** [comment]
not taking knockback allows for the knight to engage, especially against targets that it could struggle against (i.e. a medium range ranger). Yes, you could take kb on Mineplex, but I think not taking kb fits more than not dealing kb.

**Comment by ezrn** [comment]
> not taking knockback allows for the knight to engage, especially against targets that it could struggle against (i.e. a medium range ranger). Yes, you could take kb on Mineplex, but I think not taking kb fits more than not dealing kb.

bulls charge was designed to catch up to people, not dealing knockback allows it to cath up and deal significant damage to playres without them able to run away

**Comment by Mykindos** [comment]
good luck killing a ranger with knight

**Comment by StudentAlleg** [comment]
Could add it as a config option, that at least gives it an easy way to test (something like deals knockback while charging, takes knockback while charging), probably a double/100 tho

**Comment by ezrn** [comment]
> good luck killing a ranger with knight

you cant do this anyways

**Comment by ezrn** [comment]
anyways youre probobly right, ill revert it

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/sword/Sever.java [hunk 1]
```diff
@@ -98,41 +98,46 @@ private void onInteract(Player player, LivingEntity ent) {
...
         }
-        ent.getWorld().playSound(ent.getLocation(), Sound.ENTITY_SPIDER_HURT, 1.0F, 1.5F);
 
+        player.getWorld().playSound(player.getLocation(), Sound.ENTITY_SPIDER_HURT, 1.0F, 1.5F);
     }
 
     @Override
     public double getCooldown(int level) {
-        return cooldown;
+        return cooldown - cooldownDecreasePerLevel;
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/sword/Sever.java:134]: not how its calculated

should be cooldown - ((level - 1) * cooldownDecreasePerLevel)

**Review by Mykindos — CHANGES_REQUESTED** [review]

**Review by Mykindos — APPROVED** [review]

**Pull request merged**